### ISSUE TITLE
Welderbombing a hacked APC does not delay malf countdown

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#8526]
(WEB REPORT BY: yred REMOTE: 158.69.55.2:7777)

> Revision c10f8a4ec34abe8707f1c6839c8fd6acb7e00ac1 
> 
> General description of the issue
> Welderbomb directly adjacent a blue APC deletes it without increasing the time until the AI wins.
> 
> What you expected to happen
> Timer would increase substantially.
> 
> What actually happened
> Timer only continued ticking down.
> 
> Steps to reproduce if possible
> Welderbomb hacked APC, look at timer.

**Comment by gbasood** [comment]
not sure if bug or not

**Comment by ghost** [comment]
It's a bug if that's true. Destroyed APCs shouldn't count.

**Comment by 9600bauds** [comment]
Why not? You're destroying the AI's control over the station

**Comment by clusterfack** [comment]
Yeah I'm pretty sure this isn't a bug because its extremely easy to make the timer go up by a LOT this way

**Comment by 9600bauds** [comment]
So how do you make the timer go up?

**Comment by ihadtoregisterforthis** [comment]
This could lead to malf borgs welderbombing public APCs

**Comment by 9600bauds** [comment]
??????
I believe this is for BSOD APCs

**Comment by Probe1** [comment]
This applies to any deconstruction method and IS a bug.  It WAS how the game mode worked and is now broken.
